test/unit/utils/componentParserData.js

```diff
@@ -251,4 +251,58 @@ module.exports.PARSE_BOX_ICON_EXPECTED = `
 </box>
 `;
 
+/**
+ * Dropdowns
+ */
+
+module.exports.PARSE_DROPDOWN_HEADER_EXPECTED = `
```

Comment:
Small nit: perhaps move the `EXPECTED` constants below the constants they test to conform with the rest of the file

Could add deprecation notes here as well